facebook/delphiFacebook/R/contingency_variables.R

```diff
@@ -271,12 +289,143 @@ code_addl_vaccines <- function(input_data, wave) {
     input_data$v_vaccine_timing_dontknow <- NA
   }
   
+  if ("B3" %in% names(input_data)) {
+    input_data$t_taken_temp <- input_data$B3 == 1
+  } else {
+    input_data$t_taken_temp <- NA
+  }
+
   if ("B8" %in% names(input_data)) {
     input_data$t_ever_tested <- input_data$B8 == 1
   } else {
     input_data$t_ever_tested <- NA
   }
   
+  if ("B5" %in% names(input_data)) {
+    # Coded as 1 = "tested and COVID+", 2 = "tested and COVID-",
+    # 3 = "tested, but no result yet", 4 = "tried to get tested but couldn't",
+    # 5 = "didn't try to get tested"
+    # Regardless of test result
+    input_data$t_unusual_symptom_tested <- case_when(
+      input_data$B5 %in% c(1, 2, 3) ~ 1,
+      input_data$B5 %in% c(4, 5) ~ 0,
+      TRUE ~ NA_real_
+    )
+    # Tested with positive result
+    input_data$t_unusual_symptom_tested_positive <- input_data$B5 == 1
+  } else {
+    input_data$t_unusual_symptom_tested <- NA
+    input_data$t_unusual_symptom_tested_positive <- NA
+  }
+
+  if ("B6" %in% names(input_data)) {
+    input_data$t_unusual_symptom_hospital <- input_data$B6 == 1
+  } else {
+    input_data$t_unusual_symptom_hospital <- NA
+  }
+
+  if ("B7" %in% names(input_data)) {
```

Comment:
Given the issue with B7 I raised on Slack, maybe we need to throw out Wave 10 or add some logic to correct the aggregates of it; worth raising with Adrianne maybe?

Reply:
Told Adrianne about the issue.

What corrective logic are you thinking of? If B7 is being shown to only those who didn't respond at all to B2, then even if we filter it using other questions, we're not going to be able to extract any responses we're interested in (where someone answered both B2 and B2c with symptoms).

It seems to me that the only option is to drop all Wave 10 responses to B7.

Reply:
Adrianne says dropping Wave 10 sounds fine.